README.md

```diff
@@ -2,23 +2,26 @@
 Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).
 
 ## What Packages Are Available?
-1. Type `brew search` for a list.
-2. Or visit [braumeister.org](http://braumeister.org) to browse packages online.
-3. Or use `brew search --desc <keyword>` to browse packages from the command line.
+1. Use `brew search` on the command line for a list of packages.
```

Comment:
I'm a bit "eh" about the addition of "on the command line" here. _In this context_, I think the use of backtics is sufficiently clear enough, but I'm happy to bow to the change if people are actually getting confused here.